sys/kern/vfs_cache.c

```diff
@@ -190,7 +194,10 @@ TAILQ_HEAD(cache_freebatch, namecache);
  * tied to VFS SMR. Even if retaining them, the current split should be
  * re-evaluated.
  */
-#ifdef __LP64__
+#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Technically wrong for CHERI-64 but I don't care enough about that tbh here

Reply:
I think we will need further changes to various ifdefs for CHERI-64 as I would expect issues in all of those that define padding and static assertions for pcpu and mbuf stuff. We can do it but probably should be its own commit.